lib/temperature.rb

```diff
@@ -1,14 +1,19 @@
 class Temperature
+  KELVIN_FACTOR = 273.15
+
   def initialize(*args)
-    @temperature = args[0]
-    @unit = args[1]
+    @temperature = args.first
+    # @unit = args[1]
```

Comment:
@ViniciusVenancio unit is not being used, so makes no difference if it's celsius or kelvin. maybe you should make the example more simple.